omc3/scripts/merge_kmod_results.py

```diff
@@ -0,0 +1,275 @@
+"""
+Merge KMOD Results
+-------------------
+
+Script to merge the results from KMOD into one TfsDataFrame.
+The script takes the kmod-results folders as input and merges the
+lsa-result tfs-files in these together.
```

Comment:
Can be written out in a directory to serve as a measurement for global correction?
Btw, I did not see anywhere anything like "beta_kmod_", maybe I just overlooked it...

Reply:
to be done later in global correct #135